pom.xml

```diff
@@ -6,6 +6,7 @@
         <groupId>org.jboss.tools</groupId>
         <artifactId>parent</artifactId>
         <version>4.4.1.Alpha1-SNAPSHOT</version>
+        <relativePath/>
```

Comment:
That is the problem that default value is `../pom.xml` not what you think as `../` see http://maven.apache.org/ref/3.0/maven-model/maven.html

> Set the value to an empty string in case you want to disable the feature and always resolve the parent POM from the repositories.